src/common/memory/payload.cc

```diff
@@ -14,8 +14,9 @@ limitations under the License.
 */
 
 #include "common/memory/payload.h"
-
 #include <cstdint>
+#include <string>
+#include "common/util/logging.h"
```

Comment:
Group the include lines, put `#include "common/util/logging.h"` to the group of "common/..." includes above.